Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", ] as String[]
+def paths = []
+
+// using mesos node because it's a lightweight alpine docker image instead of full VM
+node('mesos') {
+  stage("Verify author") {
+    user_is_authorized(master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#tools-notify')
+  }
+}
+
+node('mesos-ubuntu') {
+  def shcmd = { String command ->
+      output = sh(script: command, returnStdout: true).trim()
+      println(output)
+      return output
+  }
+
+  checkout scm
+  stage("Get changeset") {
+    shcmd("git config remote.origin.fetch '+refs/heads/*:refs/remotes/origin/*'")
```

Comment:
We should explain the operation and why we are doing so many git-operations via  comment.

Reply:
done